h/h_api/resources/schema/bulk_api/command/configuration.json

```diff
@@ -24,7 +24,7 @@
             "properties": {
                 "total": {
                    "type": "integer",
-                    "minimum": 1,
+                    "minimum": 2,
```

Comment:
In the end we count the config instruction as an instruction, so really you should be sending us something else.